crates/protocol/src/batch/tx_data/wrapper.rs

```diff
@@ -93,6 +109,7 @@ impl SpanBatchTransactionData {
             Self::Legacy(_) => TxType::Legacy,
             Self::Eip2930(_) => TxType::Eip2930,
             Self::Eip1559(_) => TxType::Eip1559,
+            Self::Eip7702(_) => TxType::Eip1559,
```

Comment:
`Eip7702` tx type [`alloy_consensus::TxType`](https://docs.rs/alloy-consensus/0.9.2/alloy_consensus/transaction/enum.TxType.html).

```suggestion
            Self::Eip7702(_) => TxType::Eip7702,
```